ClosedXML/Excel/Cells/XLCell.cs

```diff
@@ -1877,6 +1877,12 @@ public override IEnumerable<IXLStyle> Styles
                 yield return Style;
             }
         }
+        void IXLStylized.ModifyStyle(Func<XLStyleKey, XLStyleKey> modification)
```

Comment:
Just so that I can give at least one suggestion for this PR, please add a blank line before this modification ;-)

Reply:
I knew you will notice!! :)